src/Compilers/CSharp/Test/Emit/Attributes/AttributeTests_RefReadOnly.cs

```diff
@@ -67,8 +68,11 @@ class Test
                 var parameter = method.GetParameters().Single();
                 Assert.Equal(RefKind.RefReadOnly, parameter.RefKind);
 
-                AssertSingleReadOnlyAttribute(parameter.GetAttributes(), referenceA.Compilation.AssemblyName);
-                AssertSingleReadOnlyAttribute(method.GetReturnTypeAttributes(), referenceA.Compilation.AssemblyName);
+                AssertReferencedIsReadOnlyAttribute(parameter.GetAttributes(), referenceA.Compilation.AssemblyName);
+                AssertReferencedIsReadOnlyAttribute(method.GetReturnTypeAttributes(), referenceA.Compilation.AssemblyName);
+
+                var isReadOnlyAttributeTypeName = WellKnownTypes.GetMetadataName(WellKnownType.System_Runtime_CompilerServices_IsReadOnlyAttribute);
+                Assert.Null(module.ContainingAssembly.GetTypeByMetadataName(isReadOnlyAttributeTypeName));
```

Comment:
Consider extracting a `GetIsReadOnlyAttribute(ModuleSymbol)` helper for use across the tests.